### ISSUE TITLE
Fastify: Error: Unsupported framework

### ISSUE THREAD

**Issue by gimyboya** [dougmoscrop/serverless-http#82]
As the title suggest i am getting the error `Error: Unsupported framework` when trying to use fastify as follows:

```js
// index.js
const serverless = require('serverless-http')
const app = require('./app')

module.exports.handler = serverless(app)
```
```js
// app.js
'use strict'
const fastify = require('fastify')()

// declare a new express fastify
const port = 3000
fastify.use(require('body-parser').json())
// Enable CORS for all methods
fastify.use(require('cors')())

fastify.get('/', function(request, reply) {
  reply.send({ hello: 'world' })
})

fastify.listen(port, function(err) {
  if (err) throw err
  console.info(`server listening on port: ${port}`)
})

module.exports = fastify
```

any ideas on how to fix this? as per #73 i though that `serverless-http` supports `fastify`

**Comment by dougmoscrop** [comment]
Which version of fastify? I didn't validate the support myself I took the PR at face value, would be happy to have it fixed.

**Comment by gimyboya** [comment]
@dougmoscrop currently using the latest version of `fastify` which is `v2.0.1`

**Comment by gimyboya** [comment]
Thank you for the support i really appreciate it

**Comment by dougmoscrop** [comment]
ok, yeah fastify is a lot different than what the test that was added suggests to me, at least that test required hooking in to it in a way that is not simply additive.. I am looking in to alternatives using .inject

**Comment by gimyboya** [comment]
@dougmoscrop any progress on this issue?

**Comment by duro** [comment]
@gimyboya I'm assuming this renders this lib unusable with latest Fastify?

@dougmoscrop Also curious if you have an update here?

**Comment by dougmoscrop** [comment]
not yet :/

**Comment by gimyboya** [comment]
@duro Correct so far I couldn't use it as the server won't even start

**Comment by dougmoscrop** [comment]
Should be fixed in 1.10, let me know if it doesn't work.
